test/type_prop.cpp

```diff
@@ -9731,9 +9751,9 @@ TEST(type_prop, topk_invalid_index_type)
     }
     catch (const NodeValidationFailure& error)
     {
-        EXPECT_HAS_SUBSTRING(
```

Comment:
This and several other unrelated `TopK` tests got tripped up by the change to `element::Type`. It looks like these tests are actually screwed up anyway; I'll fix them up in a separate PR.